src/content/docs/apm/agents/net-agent/configuration/net-agent-configuration.mdx

```diff
@@ -3212,14 +3212,17 @@ There are three main sub-features:
 
```

Comment:
It's hidden in the diff, but I think it should say "there are four main sub-features" now